lib/pluginOverride.js

```diff
@@ -77,3 +83,8 @@ function buildRoutePrefix (instancePrefix, pluginPrefix) {
 
   return instancePrefix + pluginPrefix
 }
+
+function loadVersion () {
+  const pkg = JSON.parse(fs.readFileSync(path.join(__dirname, '..', 'package.json')))
+  return pkg.version
```

Comment:
this should be cached.

As a suggestion I would put it in a getter on the top level fastify function (the factory).